k8s/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: k8s-web-hello
+spec:
+  type: LoadBalancer
+  selector:
+    app: k8s-web-hello
+  ports:
+  - port: 3333
+    targetPort: 3000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: k8s-web-hello
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: k8s-web-hello
+  template:
+    metadata:
+      labels:
+        app: k8s-web-hello
+    spec:
+      containers:
+      - name: k8s-web-hello
+        image: edikkkk/k8s-web-hello
+        resources:
+          limits:
+            memory: "128Mi"
+            cpu: "250m"
+        ports:
+        - containerPort: 3000
```

Comment:
keep in mind this does nothing, its not bad to put it there dont get me wrong, but this actually does nothing